manageiq-operator/deploy/crds/manageiq.org_v1alpha1_manageiq_cr.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: manageiq.org/v1alpha1
+kind: Manageiq
+metadata:
+  name: miq
+spec:
+  # Add fields here
+  appName:  "manageiq"
+  applicationAdminPassword: "smartvm"
+  applicationDomain: "miqproject.apps-crc.testing"
+
+  databaseName: "vmdb_production"
+  databasePort: "5432"
+  databaseUser: "root"
+  databasePassword: "redhat"
```

Comment:
I think these need to be `databaseSecret` now.

Reply:
Done